src/bee-module.nix

```diff
@@ -130,6 +138,39 @@
     imports = [beeModule locatedConfig extraConfig] ++ nixosModules;
   };
 
+  tranformToDarwinConfig = evaled: locatedConfig: let
+    darwinModules =
+      import (evaled.config.bee.darwin + "/modules/module-list.nix")
+      ++ [
+        evaled.options.bee.darwin.darwinModules.flakeOverrides
+      ];
+    extraConfig = {
+      # seemlessly integrate hm if desired
+      imports = l.optionals evaled.options.bee.home.isDefined [
+        evaled.config.bee.home.darwinModules.home-manager
+        {
+          home-manager.useGlobalPkgs = true;
+          home-manager.useUserPackages = true;
```

Comment:
That's likely not right / enough.

Reply:
@GTrunSec any thoughts on this one?

Reply:
Through testing, bee.nixpkgs can be correctly applied to homeModules.

Reply:
| We could but the missing bit is that system/nix/nixpkgs.nix doesn't implement nixpkgs.pkgs so dawring cannot configure pkgs entirely via the module system.
What'd be [missing](https://github.com/LnL7/nix-darwin/commit/5528b366981dab354f7ec7299c26b8c9303cdf06) is config.nixpkgs.pkgs - standalone, but without going through the eval function bootstrapping as that commit does.

- [ doesn't implement nixpkgs.pkgs] `darwinConfigurations.guangtao-macbook.config.home-manager.extraSpecialArgs.darwinConfig.nixpkgs`

- through that options that we can use the args.pkgs to config homeModule `darwinConfigurations.guangtao-macbook.config.home-manager.extraSpecialArgs.darwinConfig._module.args.pk
gs`

Reply:
own darwin-eval-config.nix ?
-  https://github.com/GTrunSec/hive-1/commit/580eadaea62b3d88723155bcd5490cbe3ef240a7